### PR TITLE
Added a C Programming Course in English

### PR DESCRIPTION
**Pull request by 5h0ov**
## What does this PR do?
Add resource(s)

## For resources
Added a C Programming Course in English.

### Why is this valuable (or not)?
Yes.

### How do we know it's really free?
It is on youtube.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by 5h0ov** [comment]
> For courses add playlist link and not individual link.

oh ok i will change it

**Comment by Ajay-singh1** [comment]
Looks Good To Me! Wait for response.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -317,6 +317,7 @@
 * [C Programming for Beginners](https://www.youtube.com/playlist?list=PL98qAXLA6aftD9ZlnjpLhdQAOFI8xIB6e) - Programiz
 * [C Programming Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWoGzOXqtKeM71OLpvZbuU0P)
 * [C Programming Tutorial for Beginners](https://www.youtube.com/watch?v=KJgsSFOSQv0) - Mike Dane (freeCodeCamp)
+* [C tutorial for beginners 🕹️](https://www.youtube.com/playlist?list=PLZPZq0r_RZOOzY_vR4zJM32SqsSInGMwe) - Bro Code
```
Comment (sohansai) [inline courses/free-courses-en.md:320]:
```suggestion
* [C tutorial for beginners](https://www.youtube.com/playlist?list=PLZPZq0r_RZOOzY_vR4zJM32SqsSInGMwe) - Bro Code
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**